app/controllers/carts_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class CartsController < ApplicationController
+  before_action :set_cart
+
+  def show
+    @cart = Cart.find(params[:id])
+  end
+
+  def create
+    @cart = Cart.new(cart_params)
+  end
+
+  def destroy
+    @cart = Cart.find(params[:id])
+    @cart.destroy
```

Comment:
sprobuj wrzucic to w ifa i daj znac czy dziala

Reply:
fixed